### DISCUSSION TITLE
What are the options to change the UI of the Backend?

### DISCUSSION THREAD

**Discussion by sjusamed** [pimcore/pimcore#15965] [Q&A / Support]
We're currently building our ecom site which will use Pimcore. 

The front end of our site is already set to be built on next.js utilizing pimcore's headless ecommerce, however, we would like to change the UI of the backend as well.

The original UI looks like it's stuck in 2005... 

We would like to know if it's possible to do so without going on the enterprise plan, and if so, what are the different options to get to our result.

We great appreciate your help guys,

**Comment by kingjia90** [comment]
In Pimcore 11, we separated the UI from the Core in an own repository/bundle https://github.com/pimcore/admin-ui-classic-bundle but, so far, there are no fully-working alternative to it yet (neither from Pimcore or "third-party"). However, it's work in progress but don't have any info to share about it.

> We would like to know if it's possible to do so without going on the enterprise plan, and if so, what are the different options to get to our result.

I don't think there would be any differences even with an enterprise plan, at the current state.

**Reply by sjusamed** [reply]
Thanks for the answer! We've looked things over and are planning on building out normally and setting up everything we will need in the regular backend for now.

In the later phase, we will try to find a way to create a react dashboard/backend and map elements accordingly to how the functionalities of the original back end. Maybe this could be done using webpack. We're having our main developer look into possible solutions. 

If we find anything, we'll post it in here. In the meantime, we greatly appreciate any feedback!

**Comment by starfruitjsc** [comment]
Guys, I'm trying to renovate Pimcore Admin, looking for your feedback on this https://demo.corepulse.io/cms/login If you guys love it, I will publish it as a bundle so anyone can use it for free :)

**Reply by fashxp** [reply]
Nice!
We are working on a new UI too and will give a first preview at the [Pimcore Inspire](https://github.com/pimcore/pimcore/discussions/16507).

Would love to collaborate then...

**Reply by starfruitjsc** [reply]
Amazing !! I would love to see it. Pls add me as tester if possible. :)

**Reply by alexuco** [reply]
love it ! Congrats for doing it !

**Reply by sjusamed** [reply]
This looks awesome!

**Reply by nwaelkens** [reply]
Awesome indeed! I love the [+] button next to an object node in the tree.

**Comment by alexuco** [comment]
@sjusamed, a current client of ours utilizing Pimcore 10 eCommerce bundle has expressed the need for a new eCommerce solution. We are currently considering your approach with a frontend with nextjs14 for this project. Is there any way you share more info ? Shall we explore some future partnership ? If interested, just let me know.

**Reply by starfruitjsc** [reply]
hey @alexuco , the approach of using NextJS for frontend, and utilizing Pimcore as headless CMS is very normal and we have already used this combo on lots of practical eCommerce projects. Just ask whatever you want or you can drop me a message via [my Linkedin](https://www.linkedin.com/in/tr%E1%BB%8Bnh-t%C3%B9ng-anh-747361b3/) or my email address anhtt@starfruit.com.vn

**Comment by drietsch** [comment]
Hi all! Great conversation going on here. As you probably know by now, we are releasing the preview of [Pimcore Studio](https://pimcore.com/en/inspire-2024/agenda-2024#pimcore-studio), the brand new UX/UI of the Pimcore interface, at the Pimcore [Inspire Developer Conference 2024](https://pimcore.com/en/inspire-2024) in March 2024. If you are coming to the Inspire, just post your UI questions, that you would like to have discussed.

On the Inspire Conference we have a special track just concerning Pimcore Studio and our vision in terms of flexibility and extendability. This also involves visual consistency and a clear design system. On the other hand we'll also have a preview of the Pimcore Studio API, so you'll get a good idea, what our crazy vision is ;-).

See ya there! Cheers, Dietz 🚀💜

**Reply by twin-elements** [reply]
Hi @drietsch  Is there any public information regarding Pimcore Studio?

**Reply by fashxp** [reply]
Hi, 
we presented at inspire and will give a first public preview access in course of April/start of May. 
In addition to that, we are also preparing a series of webinars, blog posts, etc. to keep you updated about the concepts and our progress. 
So, stay tuned... :rocket:

**Reply by twin-elements** [reply]
@fashxp Great! Can I help with the development of this package?

**Reply by fashxp** [reply]
As soon as we have the repositories public, we would love to get community support in development :smiling_face_with_three_hearts: 
We need a bit of additional time to finalize the foundations though :smile:

**Reply by twin-elements** [reply]
@fashxp Good. I will wait.

**Reply by drietsch** [reply]
Hey,

As I mentioned earlier, we showcased a first sneak peek of Pimcore Studio at the Pimcore Inspire 2024 conference. We are really eager to also release the repositories for this project. At this time, we are highly thankful for contributions—hurray! I know that some partners have already shared information that isn’t even available yet on pimcore.com. To give you a reason to grin, I might add some presentations that our Pimcore Studio team has given on the topic. In the next few days, we will also post this on pimcore.com, as we are currently preparing those posts. I hope this gives you some early insights. As @fashxp mentioned, we are still working on some foundational aspects, so it’s not yet the right time for any detailed questions. This is just to give you a quick glimpse into the world of Pimcore Studio.

For getting notified concerning the Pimcore Studio posts, etc. please subscribe to our Pimcore newsletter at [pimcore.com](https://pimcore.com) or follow us on [LinkedIn](https://www.linkedin.com/company/pimcore). 

**[Pimcore Studio Sneak Peek - Pimcore Inspire 2024](https://github.com/pimcore/pimcore/files/14945754/pimcore-inspire-studio.pdf)**

Cheers,
Dietz

**Comment by fashxp** [comment]
Hey folks, we have some news regarding UI, have a look here: https://github.com/orgs/pimcore/discussions/17076
